doc/en/capture.rst

```diff
@@ -71,7 +71,7 @@ of the failing function and hide the other one:
     =========================== test session starts ============================
     platform linux -- Python 3.x.y, pytest-4.x.y, py-1.x.y, pluggy-0.x.y
     cachedir: $PYTHON_PREFIX/.pytest_cache
-    rootdir: $REGENDOC_TMPDIR
+    rootdir: /path/to/example
```

Comment:
Those changes are due to master / https://github.com/pytest-dev/regendoc/pull/8 being used, can update from 0.6.1, but I think it is good and the mess with the missed releases is hopefully merged/fixed soon (i.e. we would get this output also with release versions).

Reply:
Anyway, we could have a custom replacement for this also..

Reply:
Added a custom replacement.